src/components/OfflineWithFeedback.tsx

```diff
@@ -57,6 +57,9 @@ type OfflineWithFeedbackProps = ChildrenProps & {
 
     /** Whether we can dismiss the error message */
     canDismissError?: boolean;
+
+    /** Whether we should render the error message above the children */
+    errorAboveChildren?: boolean;
```

Comment:
How about this name?
```suggestion
    shouldDisplayErrorAbove?: boolean;
```

Reply:
✅